packages/@glimmer/wire-format/lib/elements.ts

```diff
@@ -0,0 +1,60 @@
+export enum Tags {
```

Comment:
I can add SVG tags.Discovering is pretty difficult to do ahead of time specifically with the custom element stuff. For example we do not now AOT if `<foo-bar />` is a custom element or if it's a glimmer component.